src/client/components/questions/Address.jsx

```diff
@@ -25,12 +28,20 @@ class Address extends React.Component {
   // TODO: Look into if this is the best way to update address,
   // it is incredibly slow right now
   handleChange(path, update) {
+    // if the country is US, Canada, or Mexico, then the
+    // state/provice + postal is required
+    if (path === 'country') {
+      this.setState({ requireStateZip: _.hasIn(states, update.value) });
```

Comment:
I would prefer not to store this in state if possible. Can we instead just check the value of `this.props.value.country` directly?